### DISCUSSION TITLE
希望能够整合umi-ocr

### DISCUSSION THREAD

**Discussion by darkelfzdy** [pot-app/pot-desktop#509] [Ideas]
umi-ocr（https://github.com/hiroi-sora/Umi-OCR）是非常强大的本地ocr软件，我试了一下，准确率高，速度快，但是目前调用起来麻烦一点，希望将umi-ocr整合进内置服务中取代Windows的ocr

**Comment by darkelfzdy** [comment]
目前为了方便，我是反过来用umi-ocr调用pot-app

**Reply by sch-lda** [reply]
已经有rapidocr插件了，识别效果相同，完全没有必要将一整个umiocr整合进去

**Comment by Pylogmon** [comment]
Pot的PaddleOCR插件，使用和Umi-OCR一模一样的后端

https://github.com/pot-app/pot-app-recognize-plugin-paddle
